src/nimUtils.ts

```diff
@@ -90,26 +90,26 @@ export function prepareConfig(): void {
 export function getBinPath(tool: string): string {
     if (_pathesCache[tool]) return _pathesCache[tool];
     if (process.env['PATH']) {
-        var pathparts = (<string>process.env.PATH).split((<any>path).delimiter);
-        _pathesCache[tool] = pathparts.map(dir => path.join(dir, correctBinname(tool))).filter(candidate => fs.existsSync(candidate))[0];
```

Comment:
Why you remove these lines?
Without these lines it will not work correctly, and `which` command dont install on all linux system
I dont sure about that all version of MacOS has installed `which` too

Reply:
if any file named `nim` in $PATH 's dirs it will fails. may make more step to check

Reply:
Yes, but for windows it will not work

Reply:
so if win32 using PATH way,others `which` enough,or may check `/usr/local/bin` `/usr/bin first`